### PR TITLE
Case fan splicer

### PR DESCRIPTION
**Pull request by Tehtsuo**
* [ ] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ ] Folder and file naming:
    * Folders and filenames shouldn't contain spaces. Only letters `a-zA-Z`, numbers `0-9`, underscores `_`, hyphens `-` and periods `.`
    * Primary color: `part_xyz.stl`
    * Accent color: `[a]_part_xyz.stl`
    * Opaque color (Blocks light): `[o]_part_xyz.stl`
    * Clear/transparent color (Allows light): `[c]_part_xyz.stl`
    * Quantity, if more than one is needed: `part_xyz_x4.stl`
  * [ ] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [ ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ ] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [ ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [ ] Submitted STL files are oriented and scaled properly for printing.
  * [ ] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [ ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

Case Fan Splicer

#### How was it tested? 

Installed on my printer

#### Any background context you want to provide?

An alternative for a PCB with JST XH connectors on it.  Mounts to extrusion channel with T nuts.

#### Screenshots (if appropriate)

Installed

### CODE REVIEW

**Comment by bythorsthunder** [comment]
https://github.com/VoronDesign/VoronUsers/pull/842/commits/4336be2266359cb7ecb3600e77426f6d9f6029ee

Looks like the file name in the Voronuser Readme is Splitter and the actual file name is Case_Fan_Splicer

**Comment by Tehtsuo** [comment]
> [4336be2](https://github.com/VoronDesign/VoronUsers/commit/4336be2266359cb7ecb3600e77426f6d9f6029ee)
> 
> Looks like the file name in the Voronuser Readme is Splitter and the actual file name is Case_Fan_Splicer

Should be fixed now.

File: printer_mods/Tehtsuo/Case_Fan_Splicer/Readme.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+**Why:**
+
+Didn't feel like waiting for a PCB to be made and shipped to me, and I'd likely end up making a container for the PCB to go into anyways.
+
+![Diagram](./Installed.png)
+
+**BOM:**
+
+* 2x m3 x 12-ish
+* 2x Your choice of m3 hammer nut or m3roll in t nut
```
Comment (MapleLeafMakers) [inline printer_mods/Tehtsuo/Case_Fan_Splicer/Readme.md:18]: M3 should be capitalized throughout this file (yes, it's nitpicky, sorry)
you're missing some hyphens and spaces here. `M3 hammerhead or roll-in t-nut`

**Comment by FHeilmann** [comment]
Hi, 

I want to thank you for your submission and want to personally apologize that your submission has not gotten the review it deserves.
We are in the process to restructure the repo to streamline reviews and hopefully solve this issue in the future.

Unfortunately, this means that we will require you to resubmit the mod under the new system.

Please check [here](https://github.com/VoronDesign/VoronUsers/wiki/VoronUsers-Overhaul) for details and instructions on how to proceed. 
If you have any questions, please reach out in the #voronuser_mods channel on the VoronDesign discord.
If you have since moved on from this mod and don't want to resubmit, no further action is necessary from your side.

Bests,

Florian a.k.a. Haribro

**Pull request closed without merging**